File: spring-java/src/main/java/com/jmpark/springjava/entity/UserEntity.java
package com.jmpark.springjava.entity;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@RequiredArgsConstructor
@Getter
@ToString
@EqualsAndHashCode(callSuper = false)
public class UserEntity extends BaseTimeEntity {
    @Id
    String id = "";

    String loginId = null;
    String loginPwd = null;
    String status = null;
    String role = null;
    String name = null;
}

File: spring-kotlin/src/main/kotlin/com/jmpark/springkotlin/entity/UserEntity.kt
package com.jmpark.springkotlin.entity

import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class UserEntity (
    @Id
    val id : String = "",

    var loginId : String? = null,
    var loginPwd : String? = null,
    var status : String? = null,
    var role : String? = null,
    var name : String? = null
) : BaseTimeEntity()